### ISSUE TITLE
Add wikimedia commons tileset by Cangjie6

### ISSUE THREAD

**Issue by 5zlotych** [m4tx/chombo-gen#16]
All the tiles are licensed under CC BY 4.0
Pretty much all the different tiles are present in this tileset, this includes red 1/3/5/7/9, haku pocchi and tiles from other mahjong variants.  
https://en.wikipedia.org/wiki/Mahjong_tiles
![image](https://github.com/m4tx/chombo-gen/assets/78821076/4b315b83-2a5d-4673-9012-dac381b9dfe2)
![image](https://github.com/m4tx/chombo-gen/assets/78821076/057c49d6-edb9-41d4-b19c-7ee72a7e1ae9)
![image](https://github.com/m4tx/chombo-gen/assets/78821076/b83c8e43-fca0-44e6-b6c3-d366992a01da)
![image](https://github.com/m4tx/chombo-gen/assets/78821076/15da9b2d-30b5-4ef3-9366-42e694d5f283)

**Comment by 5zlotych** [comment]
True, however with this set it's possible to support other red tiles not supported in the generator.

**Comment by marcinkurek** [comment]
It would be interesting to see these laid out with some negative right margin to bring them closer together.

When doing that, the black outline could also be added in post-processing of the merged image instead of each individual tile, so it doesn't overlap the tile to the left.
